apps/chat/src/components/Promptbar/components/PromptbarSettings.tsx

```diff
@@ -58,6 +59,7 @@ export function PromptbarSettings() {
         Icon: IconSquareOff,
         onClick: () => {
           dispatch(PromptsActions.resetChosenPrompts());
+          dispatch(PromptsActions.resetChosenEmptyFolders());
```

Comment:
Why do we need separate action for that? We can do it into `resetChosenPrompts`

Reply:
I think it's better to implement separate actions for improved separation of concerns, testability, and reusability.

Reply:
Code changed as suggested.